x-pack/test/functional/apps/maps/geofile_wizard_auto_open.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common']);
+  const testSubjects = getService('testSubjects');
+
+  describe('Auto open file upload wizard in maps app', () => {
+    before(async () => {
+      await PageObjects.common.navigateToUrl('maps', 'map/?openLayerWizard=uploadGeoFile', {
```

Comment:
Can you update this test to click the link from integrations page. That way we know the generated URL is correct and working. There might be changes in the future that remove the appended base path and we want to know the displayed URL continues to work.